server/commands/documentPermanentDeleter.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import debug from "debug";
+import { Document, Attachment } from "../models";
+import { sequelize } from "../sequelize";
+import parseAttachmentIds from "../utils/parseAttachmentIds";
+
+export async function documentPermanentDeleter(
```

Comment:
Could we pull some of the tests across from `utils` to test the command? Specifically those around attachment handling can be fairly easily moved.

Reply:
Sure